addons/binding/org.openhab.binding.helios/ESH-INF/thing/ipvario221.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="helios" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ipvario221">
+		<label>Helios IP Vario</label>
+		<description>Helios IP Vario Door Station / Intercom with Firmware v2.21</description>
+
+		<channels>
+			<channel id="keypressed" typeId="keypressed" />
+			<channel id="keypressedstamp" typeId="keypressedstamp" />
+			<channel id="callstate" typeId="callstate" />
+			<channel id="calldirection" typeId="calldirection" />
+			<channel id="callstatestamp" typeId="callstatestamp" />
+			<channel id="card" typeId="card" />
+			<channel id="cardvalid" typeId="cardvalid" />
+			<channel id="cardstamp" typeId="cardstamp" />
+			<channel id="code" typeId="code" />
+			<channel id="codevalid" typeId="codevalid" />
+			<channel id="codestamp" typeId="codestamp" />
+			<channel id="devicestate" typeId="devicestate" />
+			<channel id="devicestatestamp" typeId="devicestatestamp" />
+			<channel id="audiolooptest" typeId="audiolooptest" />
+			<channel id="audioloopteststamp" typeId="audioloopteststamp" />
+			<channel id="motion" typeId="motion" />
+			<channel id="motionstamp" typeId="motionstamp" />
+            <channel id="noise" typeId="noise" />
+            <channel id="noisestamp" typeId="noisestamp" />
+            <channel id="keyreleased" typeId="keyreleased" />
+			<channel id="keyreleasedstamp" typeId="keyreleasedstamp" />
+			<channel id="switchstate" typeId="switchstate" />
+            <channel id="switchstateswitch" typeId="switchstateswitch" />
+            <channel id="switchstateoriginator" typeId="switchstateoriginator" />
+            <channel id="switchstatestamp" typeId="switchstatestamp" />
+		</channels>
+
+		<properties>
```

Comment:
Please remove this section here - properties are only set by the handler dynamically.